tracing-appender/Cargo.toml

```
@@ -22,7 +22,7 @@ rust-version = "1.53.0"
 
 [dependencies]
 crossbeam-channel = "0.5.6"
-time = { version = "0.3.2", default-features = false, features = ["formatting"] }
+time = { version = "0.3.2", default-features = false, features = ["formatting", "parsing"] }
```

if you take my suggestion about using the file's creation timestamp rather than parsing the date from the filename, we can remove this:
```suggestion
time = { version = "0.3.2", default-features = false, features = ["formatting"] }
```